### ISSUE TITLE
Address the inconsistencies when using Windows High Contrast Mode across whole site as well as across all components.

### ISSUE THREAD

**Issue by DBD324** [mi6/ic-design-system#91]
## Summary
Windows high contrast mode (WHCM) accounts for 4% of active Windows devices, and 30% of users with low vision use it too. The full system including the guidance site needs to be operable equally in WHCM.

## 💬 Description
Address any inconsistences by testing the whole site and all components to ensure they are operable in WHCM. 
Define improvements to component implementations to ensure operability. 
Some areas of issue discovered already include:
- Selection states on components that highlight selection (tabs, nav items, etc). 
- Hierarchy of component variants not observable. (primary, secondary, tertiary buttons are indistinguishable)

## 💰 Use value
Support WHCM will further enable the design system artefacts to be used equally by as many people as possible. 

## Additional info
Tell us anything else useful to help us understand your suggestion.

**Comment by DBD324** [comment]
Updates required:

- [ ] Focus styles: Use transparent outline instead of outline:none. 
- [ ] Buttons: Use transparent border instead of border:none. 
- [ ] Tabs: Selection indicator not appearing. Should appear as white.
- [ ] Back to top: Icon is showing in dark blue. Ideally should show as white. 
- [ ] Alerts: Add transparent border instead of 0px. 
- [ ] Classification banners: No colour so hard to make out. Add transparent border so that it is differentiated at bottom of screen. 
- [ ] Footer: Add transparent border-top to distinguish from rest of page. 
- [ ] Loading indicators: Radial and linear indeterminate loading indicators are not visible. 
- [ ] Loading indicator: Linear determinate indicator not visible. 
- [ ] Radio buttons: Selected state not visible. 
- [ ] Select, search: Focus on list item not visible when arrow keying through options. 
- [ ] Left hand nav: Selected page not visible. 
- [ ] Side navigation: Add transparent border-right. 
- [ ] Side navigation: Group items not showing as button colours. They should show as yellow not white. 
- [ ] Side navigation: Expand button not showing as button colours. The 
- [ ] Tooltips: Add transparent border. 
- [ ] Skeleton: Not visible. 
- [ ] Status tags: Add transparent border. 
- [ ] Status tags: Can't differentiate between status other than by label. Is this a problem?
- [ ] Stepper: Lots of problems. Stepper line not visible. Completed step border not visible. 
- [ ] Switch: 'On' state loses border. Add transparent border. 
- [ ] Switch: Handle not visible. Add transparent border. 
- [ ] Tabs: All items appear as selected. Can't distinguish current selected tab. 
- [ ] Nav bar: All items appear as unselected. Can't distinguish current selected nav item. 
- [ ] Anchor nav: Can't distinguish current select anchor nav item. 
- [ ] Top navigation: Add transparent border bottom.

**Comment by DBD324** [comment]
Some useful links on the topic:
https://www.smashingmagazine.com/2022/06/guide-windows-high-contrast-mode/
https://css-tricks.com/accessible-svgs-high-contrast-mode/#:~:text=To%20avoid%20using%20colors%20that%20will%20wind%20up,to%20buttonFace%205%20Background%20is%20mapped%20to%20window
https://css-tricks.com/quick-tips-for-high-contrast-mode/
https://sarahmhigley.com/writing/whcm-quick-tips/
